libclc/generic/libspirv/SOURCES

```diff
@@ -44,13 +44,10 @@ core/shared/clc_max.cl
 core/shared/clc_min.cl
 geometric/cross.cl
 geometric/distance.cl
-geometric/distance.inc
 geometric/dot.cl
 geometric/fast_distance.cl
-geometric/fast_distance.inc
```

Comment:
Why are these removed?

Reply:
We don't actually want to compile these `.inc` files to their own bytecode files, which is the primary purpose of these `SOURCES` files. Nor indeed can we - they're not complete CL files and are usually included from other files.

With the previous approach I think these were just silently ignored somehow.

I suspect someone added them for dependency purposes, but we don't need to do that anymore since we have `-MD` and pals to do the job for us.

Reply:
I figured that some include machinery was involved. Thanks for the explanation